src/repl/scala/tools/nsc/interpreter/ILoop.scala

```diff
@@ -383,7 +383,7 @@ class ILoop(in0: Option[BufferedReader], protected val out: JPrintWriter) extend
       }
   }
 
-  private def pathToPhaseWrapper = intp.originalPath("$r") + ".phased.atCurrent"
```

Comment:
`s/definesTopLevel/isIObjectWrapper` throughout as the policy of when to pick one wrapper style over another might change and the code using this boolean only needs to know the what, not the why.